flink-core/src/test/java/org/apache/flink/api/common/ExecutionConfigTest.java

```diff
@@ -297,7 +294,7 @@ private void testLoadingSchedulerTypeFromConfiguration(
         configFromConfiguration.configure(
                 configuration, Thread.currentThread().getContextClassLoader());
 
-        assertThat(configFromConfiguration.getSchedulerType().get(), is(schedulerType));
```

Comment:
This hotfix should happen before all the changes. And that we do not change something over and over.